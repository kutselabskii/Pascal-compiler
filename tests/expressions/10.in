not (not(not 1) + 2) = (2 and 3)