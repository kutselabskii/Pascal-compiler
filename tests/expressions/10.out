|__ equal
    |-- bitwise_negation
    |   |-- addition
    |   |   |-- bitwise_negation
    |   |   |   |-- bitwise_negation
    |   |   |   |   |-- 1
    |   |   |__ 2
    |__ bitwise_and
        |-- 2
        |__ 3
